oracle/Assets/Scripts/DelphiClient.cs

```diff
@@ -43,14 +43,22 @@ void Awake()
     }
 
 
+    /// <summary>
+    /// Updates all of the fields for each of the rooms supplied.
+    /// </summary>
+    public void ToggleRoomStatus(Room room)
+    {
+        StartCoroutine(CallToggleStatus(room));
+    }
+
     /// <summary>
     /// Updates all of the fields for each of the rooms supplied.
     /// </summary>
     public void UpdateRooms(Room[] rooms)
     {
         int[] roomIds = rooms.Select(room => room.Id).ToArray();
         StartCoroutine(CallGetCounts(roomIds, (response) => rooms.ToList().ForEach(room => room.PeopleCount = response[room.Id])));
-        // todo: update room status
+        StartCoroutine(CallGetStatuses(roomIds, (response) => rooms.ToList().ForEach(room => room.Status = response[room.Id])));
```

Comment:
room status is now updated alongside status (in .5 second intervals)

Reply:
is this for getting data from delphi or sending data to delphi?

Reply:
gets the statuses. meant to say update local room statuses